eng/versioning/external_dependencies.txt

```diff
@@ -306,7 +306,7 @@ cosmos_org.apache.spark:spark-hive_2.12;3.1.1
 cosmos_org.scala-lang:scala-library;2.12.10
 cosmos_org.scala-lang.modules:scala-java8-compat_2.12;0.8.0
 cosmos_io.projectreactor:reactor-scala-extensions_2.12;0.8.0
-cosmos_commons-io:commons-io;2.4
+cosmos_commons-io:commons-io;2.7
```

Comment:
Cosmos DB SDK relies on commons-io:commons-io and that's perfectly fine to upgrade that one.

However, cosmos_commons-io:commons-io, is not used by the cosmos db SDK, but it is used by cosmos db spark connector, and as you see in the cosmos db spark connector this is a provided dependency. this being a provided dependency, cosmos db spark connector is not bringing in commons-io, but it will be provided by the spark platform.

You can see here: https://docs.databricks.com/release-notes/runtime/9.1.html that spark platform is still relying on commons-io 2.4 version.

we should revert this change for two reasons:
1. doesn't help anyone, as this is a provided dependency and at the end it is spark platform which is bringing in commons-io lib
2. this actually introduces a risk, as we may develop or test something on a library which is not the same as what is present on the spark platform.

Reply:
reverted this one.